app/businessrates/authorisation/repositories/AccountsCache.scala

```diff
@@ -67,5 +72,6 @@ private[repositories] case class Record(_id: String, data: Accounts, createdAt:
 
 private[repositories] object Record {
 
+  private implicit val dateFormat = localDateTimeFormat
```

Comment:
i assume this will work because the code recovers from errors accessing the accountsCache? Because the String timestamps we have currently stored in mongo will fail to read back in using the new formatter.